src/main/java/mate/hwdao/controllers/AddDriverToCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.hwdao.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.hwdao.Main;
+import mate.hwdao.lib.Injector;
+import mate.hwdao.model.Car;
+import mate.hwdao.model.Driver;
+import mate.hwdao.service.CarService;
+import mate.hwdao.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+
```

Comment:
Don’t begin class or method implementation with an empty line

Reply:
that's just your point of view, google style is said: A single blank line may also appear anywhere it improves readability, for example between statements to organize the code into logical subsections.

Reply:
It is up to you) I am not up to something of my personal preferences, it is just a matter of vigilance.